### ISSUE TITLE
MariaDB in documentation

### ISSUE THREAD

**Issue by robertsilen** [Icinga/icingaweb2#5237]
Sorry for not using the default template for a Bug/Feature, but this is an improvement suggestion for the documentation of icingaweb2. 

I noticed the [installation docs](https://icinga.com/docs/icinga-web/2.8/doc/02-Installation/) ([file in github](https://github.com/Icinga/icingaweb2/edit/main/doc/02-Installation.md)) only mentions MySQL in the requirements, but the actual terminal code example uses MariaDB. Would it be fair to add MariaDB to the requirements, perhaps even as recommended database? Shall I do a PR for it? 

This is part of my efforts I describe in my blog [Improving MariaDB Support in Open Source Projects](https://mariadb.org/improving-mariadb-support-in-open-source-projects/).

**Comment by nilmerg** [comment]
yes, we should, I’ll approve a PR :stuck_out_tongue_winking_eye:
